application/modules/home/controllers/Home.php

```diff
@@ -43,4 +43,12 @@ public function index() {
 		// load view
 		$this->load_view('top', $this->data);
 	}
+
+	public function side_ci()
+	{
+		$this->data["exhibition"]["name"] = "test";
+
+		 // load view
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed for code indentation

<!-- Comment by [SideCI](https://sideci.com) -->